src/Nether.Analytics/FileOutputManager.cs

```diff
@@ -72,19 +72,42 @@ private string GetFilePath(string pipelineName, int idx, Message msg)
                     if (File.Exists(filePath))
                     {
                         await AppendMessageToFileAsync(serializedMessage, filePath);
-
                         tryAgain = false;
                     }
                     else
                     {
                         Directory.CreateDirectory(Path.GetDirectoryName(filePath));
 
+                        // append the header to the file
                         string header = $"{_serializer.Header}{Environment.NewLine}";
-
                         await AppendMessageToFileAsync(header, filePath);
                     }
                 }
-                catch (Exception)
+                catch (Exception e)
```

Comment:
If you don't use the variable e, then don't declare it. The previous implementation was better and didn't cause any warnings.